wolox-books/src/app/components/custom-validations/confirm.password.ts

```diff
@@ -3,12 +3,12 @@ import {AbstractControl} from '@angular/forms';
 export class ConfirmPassword {
   static MatchPassword(AC: AbstractControl) {
     let password = AC.get('password').value;
-    let confirmPassword = AC.get('confirmPassword').value;
+    let confirmPassword = AC.get('confirm_password').value;
```

Comment:
Why are the variables now snake_case and not camelCase? 🤔

Reply:
Changed variables name so they match JSON request keys